vumi/scripts/vumi_model_migrator.py

```diff
@@ -186,12 +189,15 @@ def migrate_all_keys(self, continuation=None):
             continuation=continuation)
         yield self.migrate_pages(index_page, emit_progress)
 
-    def run(self):
+    def _run(self):
         if self.options["keys"] is not None:
             return self.migrate_specified_keys(self.options["keys"].split(","))
         else:
             return self.migrate_all_keys(self.options["continuation-token"])
 
+    def run(self):
+        return self._run().addBoth(lambda _: self.cleanup())
```

Comment:
Probably because I wrote them at different times. I'll change them to be consistent -- do you have any preference?